blog/_posts/2021-05-18-argocd-private-clusters.md

```diff
@@ -0,0 +1,419 @@
+---
+layout: post
+title: Learn how to manage apps across private Kubernetes clusters
+description: Learn a simple approach to deploying applications across multiple public and private Kubernetes clusters using ArgoCD, inlets PRO and GitOps.
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud multi-cloud gitops argocd
+author_img: jsiebens
+image: /images/2021-05-18-argocd-private-clusters/background.jpg
+date: 2021-05-18
+---
+
+In this post, you'll learn how to apply GitOps on multiple private Kubernetes cluster with a single Argo CD installation.
+
+## Continuous Deployment on Kubernetes with GitOps
```

Comment:
This whole section is a lot for a reader to get through.

What if you started with a brief introduction about what the reader will get in some bullets before going into depth about GitOps?